tests/test_cli.py

```diff
@@ -0,0 +1,92 @@
+import os
+from pathlib import Path
+from sabres.cli import main
+from sabres import merge_sabres
+import pytest
+
+
+
+@pytest.mark.parametrize(
+    "options,expected", 
+    [
+        (["--help"], "show this help message and exit"),
+        (["-h"], "show this help message and exit"),
+        (["--version"], "SABRes v"),
+])
+def test_helpers(capsys, options, expected):
+    try:
+        main(options)
+    except SystemExit:
+        pass
+    
+    result = capsys.readouterr().out
+    assert expected in result
+
+
+@pytest.mark.parametrize(
+    "options,expected", 
+    [
+        (["--vcall"], "argument --vcall/-v: expected one argument"),
+        ([], "the following arguments are required: --vcall/-v, --input/-i"),
+        (["--vcall", "ivar"], "the following arguments are required: --input/-i"),
+        (["--vcall", "ivar", "--outdir", 'outdir'], "the following arguments are required: --input/-i"),
+])
+def test_missing_args(capsys, options, expected):
+    try:
+        main(options)
+    except SystemExit:
+        pass
+    result = capsys.readouterr().err
+    assert expected in result
+
+
+@pytest.fixture()
+def out_dir(OUTDIR = 'tests/outdir/'):
+    try:
+        os.mkdir(OUTDIR)
+    except FileExistsError:
+        pass
+    yield OUTDIR
```

Comment:
This is the correct use of yield but this might be better off using the tempfile library https://docs.python.org/3.10/library/tempfile.html

Also the full-case implies a "constant" value (usually globally constant to the module) but it clearly isn't here  https://peps.python.org/pep-0008/#constants